sample_codes/hello_world.cpp

```diff
@@ -0,0 +1,6 @@
+#include <iostream>
```

Comment:
WARNING:  No copyright message found.  You should have a line: "Copyright [year] <Copyright Owner>"  [legal/copyright] [5]